plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/catalog/glue/IcebergGlueCatalogModule.java

```diff
@@ -56,11 +56,10 @@ protected void setup(Binder binder)
         binder.bind(TrinoCatalogFactory.class).to(TrinoGlueCatalogFactory.class).in(Scopes.SINGLETON);
         newExporter(binder).export(TrinoCatalogFactory.class).withGeneratedName();
 
-        Multibinder<RequestHandler2> requestHandlers = newSetBinder(binder, RequestHandler2.class, ForGlueHiveMetastore.class);
```

Comment:
the part is not needed: `Multibinder<RequestHandler2> requestHandlers =` 
but this  `newSetBinder(binder, RequestHandler2.class, ForGlueHiveMetastore.class);`  probably is... i assume the `@ForGlueHiveMetastore Set<RequestHandler2>` needs to be declared in Guice even when no binds are added.

Reply:
i see now that the set binder is unconditionally declared in GlueMetastoreModule, so can be skipped here